internal/fixtures/automation_configs.go

```diff
@@ -258,3 +260,35 @@ func EmptyAutomationConfig() *opsmngr.AutomationConfig {
 		ReplicaSets: make([]*opsmngr.ReplicaSet, 0),
 	}
 }
+
+func Indexes() []*map[string]interface{} {
+
+	return []*map[string]interface{}{
+		{
+			"dbName":         "dbname",
+			"collectionName": "collection",
+			"rsName":         "name",
+			"keys": []map[string]interface{}{
```

Comment:
the marshal and unmarshal of a `[]map[string]interface{} `creates a `[]interface{}` which has inside a `[]map[string]interface{}`. I do not know how to change this to include ` []interface{}`